cpu.h

```diff
@@ -0,0 +1,6 @@
+#include "gameboy.h"
+
+uint8_t __gb_execute_cb(struct gb_s *gb);
+
+/* Internal function used to step the CPU */
+void __gb_step_cpu(struct gb_s *gb);
```

Comment:
Both and `__gb_execute_cb ` and `____gb_step_cpu` are internal functions, and you should keep them "static."
The `cpu.h` should provide the public function `gb_frame`, which calls the above static functions.